File: pset1/mario/less/mario.c
// MARIO LESS

#include <stdio.h>
#include <cs50.h>

int main()
{
    // Variable initialisation
    int height;

    // Input validation
    do
    {
        height = get_int("Height of pyramid (0-23): ");
    }
    while (height < 0 || height > 23);

    // Printing part
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height + 1; j++)
        {
            if (j < height - i - 1) printf(" ");
            else printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: pset6/Sentimental/Mario/more/mario.py
from cs50 import *

height = get_int('Height: ')
while height < 1 or height > 23:
    height = get_int('Height: ')

layer = ""

for i in range(height):
    for j in range(height - 1 - i):
        layer += " "

    for k in range(2):
        for j in range(i + 1):
            layer += "#"

        layer += "  "

    print(layer)
    layer = ""
